terraform/integration/main.tf

```diff
@@ -31,8 +31,8 @@ locals {
   multi_az         = false
   application_port = 8080
 
-  app_host                  = "integration.register-home-to-rent.communities.gov.uk"
-  load_balancer_domain_name = "integration.lb.register-home-to-rent.communities.gov.uk"
+  app_host                  = "integration.register-home-to-rent.test.communities.gov.uk"
```

Comment:
particularly with a squash-merge workflow, I think my ideal for changes like this would be separate small PRs that make it into the final git history, rather than a multi-feature PR title. 

Not a big deal and not worth unpicking this one over though!

Reply:
Yep, I normally would have done that, except I had to `apply` the changes to generate new domains for the SN request and only then realised I'd added them to the wrong branch🙃. And as you say it didn't quite seem worth unpicking for this case.